templates/detail.jinja2

```diff
@@ -0,0 +1,23 @@
+{% extends "base.jinja2" %}
+{% block body %}
+  <hr>
+  <h2 id="gallery">Detailed Post View</h2>
+  <hr>
+    {% for entry in entries %}
+        <article class="entry" id="title={{ entry.id }}">
+         <h3>{{ entry.title }}</h3>
+          <div class="post">
+             <p class="dateline">{{ entry.created.strftime('%b. %d, %Y') }}</p>
+              <div class="entry_body" id ="detail_post">
+               {{ entry.text|markdown|safe }}
```

Comment:
It looks like you're using the filter here, but you are also rendering the text to html above in the view function.  Twice rendered does not make your markdown twice as awesome.